--- ./DSS-4_1_2.src/Buildit.orig	2002-11-02 02:08:06.000000000 +0100
+++ DSS-4_1_2.src/Buildit	2003-12-28 14:58:36.000000000 +0100
@@ -16,17 +16,17 @@
 
 	Linux.ppc)
         echo "Configuring for the "$OSNAME" "$HARDWARENAME" platform"
-		CPLUS=gcc
-		CCOMP=gcc
-		LINKER='gcc'
+		CPLUS=$CXX
+		CCOMP=$CXX
+		LINKER=$CXX
  		MAKE=make
  		
-		COMPILER_FLAGS= "-D__linuxppc__ -Wno-multichar -pipe"
+		COMPILER_FLAGS= "-fPIC -D__linuxppc__ -Wno-multichar -pipe"
         INCLUDE_FLAG="-include"
 		
 		CORE_LINK_LIBS="-lpthread -ldl -lm -lcrypt"
 
-		SHARED=-shared
+		SHARED="-fPIC -shared"
 		MODULE_LIBS=
 		
 		if [ -f /usr/include/socketbits.h ]; then
@@ -38,17 +38,17 @@
 	Linux.i586 | \
 	Linux.i686)
         echo "Configuring for the "$OSNAME" "$HARDWARENAME" platform"
-		CPLUS=gcc
-		CCOMP=gcc
-		LINKER='gcc'
+		CPLUS=$CXX
+		CCOMP=$CXX
+		LINKER=$CXX
  		MAKE=make
  		
-		COMPILER_FLAGS="-D_REENTRANT -D__USE_POSIX -D__linux__ -pipe"
+		COMPILER_FLAGS="-fPIC -D_REENTRANT -D__USE_POSIX -D__linux__ -pipe"
         INCLUDE_FLAG="-include"
 		
 		CORE_LINK_LIBS="-lpthread -ldl -lm -lcrypt -lstdc++"
 
-		SHARED=-shared
+		SHARED="-fPIC -shared"
 		MODULE_LIBS=
 		
 		if [ -f /usr/include/socketbits.h ]; then
@@ -135,19 +135,19 @@
 
 	
 	*)
-		echo "I don't know your platform. I'll assume this is a Linux x86 platform."
+		echo "I don't know your platform. I'll assume this is a Linux platform."
 		echo "Please edit the BuildServer script & PlatformHeader.h to add your platform."
-		CPLUS=gcc
-		CCOMP=gcc
-		LINKER='egcs'
+		CPLUS=$CXX
+		CCOMP=$CXX
+		LINKER=$CXX
  		MAKE=make
  		
-		COMPILER_FLAGS=-D__linux__
+		COMPILER_FLAGS="-fPIC -D_REENTRANT -D__USE_POSIX-D__linux__"
         INCLUDE_FLAG="-include"
 
 		CORE_LINK_LIBS="-lpthread -ldl -lm -lcrypt"
 
-		SHARED=-shared
+		SHARED="-fPIC -shared"
 		MODULE_LIBS=
 		
 		if [ -f /usr/include/socketbits.h ]; then
